opencensus/trace/config_integration.py

```diff
@@ -17,30 +17,23 @@
 
 log = logging.getLogger(__name__)
 
-SUPPORTED_INTEGRATIONS = {
-    'httplib': 'opencensus.trace.ext.httplib.trace',
-    'requests': 'opencensus.trace.ext.requests.trace',
-    'google_cloud_clientlibs': 'opencensus.trace.ext.google_cloud_clientlibs.trace',  # noqa
-    'threading': 'opencensus.trace.ext.threading.trace',
+SUPPORTED_INTEGRATIONS = ['httplib', 'mysql', 'postgresql', 'pymysql',
```

Comment:
@c24t I wish to know your opinion on this.

This "config integration" seems to be introduced for:
1. Frameworks such like Django which prefers configuration.
2. Convenience, e.g. having settings.py file (or even environment variable) which can be updated easily.

From dependency perspective, this is like "reflection", the core package is "depending" on contrib packages.

Reply:
I would prefer to keep this feature, but refactor in the following way:
1. remove hard-coded integration names.
2. scan `opencensus.ext` namespace, try to import the extensions.
3. log warning if module not found.

Reply:
That sounds like a good plan to me, although scanning installed packages may introduce some problems. Another complicating factor is that not every integration implements `trace.trace_integration`.